generators/server/files.js

```diff
@@ -178,7 +178,7 @@ const couchbaseFiles = {
       templates: [
         {
           file: 'package/repository/N1qlCouchbaseRepositoryTest.java',
-          renameTo: generator => `${generator.testDir}repository/N1qlCouchbaseRepositoryTest.java`,
+          renameTo: generator => `${generator.testDir}repository/${generator.reactive ? 'Reactive' : ''}N1qlCouchbaseRepositoryTest.java`,
```

Comment:
Same as above, keep same filename for reactive/non reactive
```suggestion
          renameTo: generator => `${generator.testDir}repository/N1qlCouchbaseRepositoryTest.java`,
```